parser.js

```diff
@@ -73,6 +73,7 @@ const metadataRules = {
       ['meta[property="og:image:secure_url"]', node => node.element.getAttribute('content')],
       ['meta[property="og:image:url"]', node => node.element.getAttribute('content')],
       ['meta[property="og:image"]', node => node.element.getAttribute('content')],
+      ['meta[name="twitter:image"]', node => node.element.getAttribute('content')],
```

Comment:
As discussed in #40, I added the syntactically correct `meta[name="twitter:*"]`, but kept the existing `meta[property="twitter:*"]` as a fallback since it seems people are fallible and make mistakes/typos.